### DISCUSSION TITLE
Frigate always restarts on a fresh debian 12 installation

### DISCUSSION THREAD

**Discussion by cesa030** [blakeblackshear/frigate#14498] [General]
Hello everyone,

I've been trying to get frigate to work for a few days now. I would like to use a Lenovo E135 for this.
Most recently, I used this guide (https://docs.frigate.video/guides/getting_started/) and followed all the steps accordingly. My setup directories look exactly like the instructions. I didn't create a config.yml because I wanted to edit it in the built-in configuration editor. After I start docker, I can log in to frigate from time to time. However, the configuration editor cannot be opened. And then I get the error message “Website cannot be reached”. 

A few days ago I installed Linux Ubuntu 22.04 and tried to get frigate to work. But I got the same error. 

What did i do wrong? Thanks for the help in advance!

Frigate logs:

> s6-rc: info: service s6rc-fdholder: starting
s6-rc: info: service s6rc-oneshot-runner: starting
s6-rc: info: service s6rc-oneshot-runner successfully started
s6-rc: info: service fix-attrs: starting
s6-rc: info: service s6rc-fdholder successfully started
s6-rc: info: service fix-attrs successfully started
s6-rc: info: service legacy-cont-init: starting
s6-rc: info: service legacy-cont-init successfully started
s6-rc: info: service log-prepare: starting
s6-rc: info: service log-prepare successfully started
s6-rc: info: service nginx-log: starting
s6-rc: info: service go2rtc-log: starting
s6-rc: info: service frigate-log: starting
s6-rc: info: service certsync-log: starting
s6-rc: info: service nginx-log successfully started
s6-rc: info: service go2rtc-log successfully started
s6-rc: info: service go2rtc: starting
s6-rc: info: service frigate-log successfully started
s6-rc: info: service go2rtc successfully started
s6-rc: info: service go2rtc-healthcheck: starting
s6-rc: info: service frigate: starting
s6-rc: info: service certsync-log successfully started
s6-rc: info: service go2rtc-healthcheck successfully started
s6-rc: info: service frigate successfully started
s6-rc: info: service nginx: starting
2024-10-22 04:05:33.885642536  [INFO] Preparing Frigate...
2024-10-22 04:05:33.906271760  [INFO] Preparing new go2rtc config...
2024-10-22 04:05:33.914643692  [INFO] Starting NGINX...
2024-10-22 04:05:33.927700208  [INFO] Starting Frigate...
s6-rc: info: service nginx successfully started
s6-rc: info: service certsync: starting
s6-rc: info: service certsync successfully started
s6-rc: info: service legacy-services: starting
2024-10-22 04:05:35.454874276  [INFO] Starting certsync...
s6-rc: info: service legacy-services successfully started
2024-10-22 04:05:35.819631404  [INFO] Starting go2rtc...
2024-10-22 04:05:36.212585044  127.0.0.1 - - [22/Oct/2024:04:05:36 +0000] "" 400 0 "-" "-" "-"
2024-10-22 04:05:36.245181364  04:05:36.244 INF go2rtc platform=linux/amd64 revision=b2399f3 version=1.9.2
2024-10-22 04:05:36.245889518  04:05:36.245 INF config path=/dev/shm/go2rtc.yaml
2024-10-22 04:05:36.248082911  04:05:36.247 INF [rtsp] listen addr=:8554
2024-10-22 04:05:36.248754085  04:05:36.247 INF [api] listen addr=:1984
2024-10-22 04:05:36.253741662  04:05:36.253 INF [webrtc] listen addr=:8555/tcp
2024-10-22 04:05:37.632843946  [INFO] Service Frigate exited with code 256 (by signal 4)
s6-rc: info: service legacy-services: stopping
s6-rc: info: service legacy-services successfully stopped
s6-rc: info: service go2rtc-healthcheck: stopping
s6-rc: info: service certsync: stopping
2024-10-22 04:05:37.680779376  [INFO] The go2rtc-healthcheck service exited with code 256 (by signal 15)
s6-rc: info: service go2rtc-healthcheck successfully stopped
2024-10-22 04:05:37.689216427  [INFO] Service CERTSYNC exited with code 256 (by signal 15)
s6-rc: info: service certsync successfully stopped
s6-rc: info: service nginx: stopping
s6-rc: info: service certsync-log: stopping
s6-rc: info: service certsync-log successfully stopped
2024-10-22 04:05:37.768611427  [INFO] Service NGINX exited with code 0 (by signal 0)
s6-rc: info: service nginx successfully stopped
s6-rc: info: service nginx-log: stopping
s6-rc: info: service frigate: stopping
s6-rc: info: service frigate successfully stopped
s6-rc: info: service go2rtc: stopping
s6-rc: info: service frigate-log: stopping
s6-rc: info: service nginx-log successfully stopped
2024-10-22 04:05:37.786793188  exit with signal: terminated
s6-rc: info: service frigate-log successfully stopped
2024-10-22 04:05:37.803827359  [INFO] The go2rtc service exited with code 0 (by signal 0)
s6-rc: info: service go2rtc successfully stopped
s6-rc: info: service go2rtc-log: stopping
s6-rc: info: service go2rtc-log successfully stopped
s6-rc: info: service log-prepare: stopping
s6-rc: info: service s6rc-fdholder: stopping
s6-rc: info: service log-prepare successfully stopped
s6-rc: info: service legacy-cont-init: stopping
s6-rc: info: service s6rc-fdholder successfully stopped
s6-rc: info: service legacy-cont-init successfully stopped
s6-rc: info: service fix-attrs: stopping
s6-rc: info: service fix-attrs successfully stopped
s6-rc: info: service s6rc-oneshot-runner: stopping
s6-rc: info: service s6rc-oneshot-runner successfully stopped
s6-rc: info: service s6rc-fdholder: starting
s6-rc: info: service s6rc-oneshot-runner: starting
s6-rc: info: service s6rc-oneshot-runner successfully started
s6-rc: info: service fix-attrs: starting
s6-rc: info: service s6rc-fdholder successfully started
s6-rc: info: service fix-attrs successfully started
s6-rc: info: service legacy-cont-init: starting
s6-rc: info: service legacy-cont-init successfully started
s6-rc: info: service log-prepare: starting
s6-rc: info: service log-prepare successfully started
s6-rc: info: service nginx-log: starting
s6-rc: info: service go2rtc-log: starting
s6-rc: info: service frigate-log: starting
s6-rc: info: service certsync-log: starting
s6-rc: info: service nginx-log successfully started
s6-rc: info: service certsync-log successfully started
s6-rc: info: service go2rtc-log successfully started
s6-rc: info: service go2rtc: starting
s6-rc: info: service frigate-log successfully started
s6-rc: info: service go2rtc successfully started
s6-rc: info: service go2rtc-healthcheck: starting
s6-rc: info: service frigate: starting
s6-rc: info: service frigate successfully started
s6-rc: info: service nginx: starting
s6-rc: info: service go2rtc-healthcheck successfully started
2024-10-22 04:06:34.491782652  [INFO] Preparing Frigate...
2024-10-22 04:06:34.502196321  [INFO] Starting NGINX...
2024-10-22 04:06:34.512245985  [INFO] Preparing new go2rtc config...
2024-10-22 04:06:34.540448142  [INFO] Starting Frigate...
s6-rc: info: service nginx successfully started
s6-rc: info: service certsync: starting
s6-rc: info: service certsync successfully started
s6-rc: info: service legacy-services: starting
2024-10-22 04:06:36.043018417  [INFO] Starting certsync...
s6-rc: info: service legacy-services successfully started
2024-10-22 04:06:36.431628575  [INFO] Starting go2rtc...
2024-10-22 04:06:36.713211195  127.0.0.1 - - [22/Oct/2024:04:06:36 +0000] "" 400 0 "-" "-" "-"
2024-10-22 04:06:36.763735277  04:06:36.763 INF go2rtc platform=linux/amd64 revision=b2399f3 version=1.9.2
2024-10-22 04:06:36.763906375  04:06:36.763 INF config path=/dev/shm/go2rtc.yaml
2024-10-22 04:06:36.766135655  04:06:36.765 INF [rtsp] listen addr=:8554
2024-10-22 04:06:36.766929687  04:06:36.766 INF [api] listen addr=:1984
2024-10-22 04:06:36.771233232  04:06:36.771 INF [webrtc] listen addr=:8555/tcp
2024-10-22 04:06:38.239851075  192.168.3.66 - - [22/Oct/2024:04:06:38 +0000] "GET / HTTP/1.1" 200 2651 "-" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/129.0.0.0 Safari/537.36" "-"
2024-10-22 04:06:38.389861291  [INFO] Service Frigate exited with code 256 (by signal 4)
s6-rc: info: service legacy-services: stopping
s6-rc: info: service legacy-services successfully stopped
s6-rc: info: service go2rtc-healthcheck: stopping
s6-rc: info: service certsync: stopping
2024-10-22 04:06:38.443911978  [INFO] Service CERTSYNC exited with code 256 (by signal 15)
s6-rc: info: service certsync successfully stopped
s6-rc: info: service nginx: stopping
s6-rc: info: service certsync-log: stopping
2024-10-22 04:06:38.453607669  [INFO] The go2rtc-healthcheck service exited with code 256 (by signal 15)
s6-rc: info: service go2rtc-healthcheck successfully stopped
s6-rc: info: service certsync-log successfully stopped
2024-10-22 04:06:38.520555822  [INFO] Service NGINX exited with code 0 (by signal 0)
s6-rc: info: service nginx successfully stopped
s6-rc: info: service nginx-log: stopping
s6-rc: info: service frigate: stopping
s6-rc: info: service frigate successfully stopped
s6-rc: info: service go2rtc: stopping
s6-rc: info: service frigate-log: stopping
s6-rc: info: service nginx-log successfully stopped
2024-10-22 04:06:38.537888472  exit with signal: terminated
s6-rc: info: service frigate-log successfully stopped
2024-10-22 04:06:38.555170682  [INFO] The go2rtc service exited with code 0 (by signal 0)
s6-rc: info: service go2rtc successfully stopped
s6-rc: info: service go2rtc-log: stopping
s6-rc: info: service go2rtc-log successfully stopped
s6-rc: info: service log-prepare: stopping
s6-rc: info: service s6rc-fdholder: stopping
s6-rc: info: service log-prepare successfully stopped
s6-rc: info: service legacy-cont-init: stopping
s6-rc: info: service s6rc-fdholder successfully stopped
s6-rc: info: service legacy-cont-init successfully stopped
s6-rc: info: service fix-attrs: stopping
s6-rc: info: service fix-attrs successfully stopped
s6-rc: info: service s6rc-oneshot-runner: stopping
s6-rc: info: service s6rc-oneshot-runner successfully stopped

docker-compose.yml
> 

> version: "3.9"

services:
  frigate:
    container_name: frigate
    restart: unless-stopped
    image: ghcr.io/blakeblackshear/frigate:stable
    volumes:
      - ./config:/config
      - ./storage:/media/frigate
      - type: tmpfs # Optional: 1GB of memory, reduces SSD/SD Card wear
        target: /tmp/cache
        tmpfs:
          size: 1000000000
    ports:
      - "8971:8971"
      - "8554:8554" # RTSP feeds

**Comment by NickM-27** [comment]
what is your config file?

**Reply by cesa030** [reply]
I didn't create a config file. The [tutorial](https://docs.frigate.video/guides/getting_started/) says: 

> You can see this by running docker logs frigate. Frigate should now be accessible at https://server_ip:8971 where you can login with the admin user and finish the configuration using the built-in configuration editor.

**Reply by NickM-27** [reply]
oh right, it should create a default config file for you. Seems that might have failed

**Reply by cesa030** [reply]
Thanks, that's a good tip. I'll try to create a simple config.yml. I can see that no such file has been created in the config directory yet.

**Reply by cesa030** [reply]
I've tried different things now. I see the config folder being created when it doesn't already exist. However, the config.yml is not created in the folder itself. I tried to create a config.yml manually with the following content:

> mqtt:
>   enabled: False
> 
> cameras:
>   dummy_camera: # <--- this will be changed to your actual camera later
>     enabled: False
>     ffmpeg:
>       inputs:
>         - path: rtsp://127.0.0.1:554/rtsp
>           roles:
>             - detect

I still get the same error message.

**Reply by NickM-27** [reply]
where are you running the docker command from?

**Reply by cesa030** [reply]
I'm running the docker command in home/USER/ .
The structure of the folder looks like this:

![Screenshot 2024-10-22 155620](https://github.com/user-attachments/assets/6c4aa08e-a302-4290-9718-ed30bb9d9266)

**Reply by NickM-27** [reply]
What are the contents of config

**Reply by cesa030** [reply]
The config folder is empty

**Reply by cesa030** [reply]
I created the structure exactly as described in the instructions

![Screenshot 2024-10-23 090648](https://github.com/user-attachments/assets/0437f538-89ef-4b8d-9484-216c09ee7f1e)

**Reply by NickM-27** [reply]
right, did you install docker exactly the way the docs say to? I have followed these instructions multiple times recently and it worked without issue. In any case if the config directory is empty you might try adding the dummy config file inside there to see if that fixes the issue.

**Reply by cesa030** [reply]
I tried it again on another PC and now it works perfectly. Apparently something really went wrong with the Docker. 

Thanks for the support.
